src/arithmetic.jl

```diff
@@ -205,6 +219,48 @@ end
 -(a::Taylor1{T}, b::TaylorN{S}) where {T<:NumberNotSeries,S<:NumberNotSeries} =
     -(promote(a,b)...)
 
+@inline +(a::STaylor1{N,T}, b::STaylor1{N,T}) where {N, T<:Number} = STaylor1(add_tuples(a.coeffs, b.coeffs))
```

Comment:
Can you just use `STaylor1(`a.coeffs .+ b.coeffs`)`?

Reply:
Done!